pkg/base/config.go

```diff
@@ -577,7 +579,14 @@ func (cfg *RaftConfig) SetDefaults() {
 		cfg.RaftMaxInflightMsgs = defaultRaftMaxInflightMsgs
 	}
 	if cfg.RaftMaxInflightBytes == 0 {
-		cfg.RaftMaxInflightBytes = uint64(defaultRaftMaxInflightBytes)
+		maxInflight := uint64(defaultRaftMaxInflightBytes)
+		if maxInflight == 0 {
+			// TODO(#90314): lower this limit to something close to max rates observed
+			// in healthy clusters. Currently this value is a conservatively large
+			// multiple so that it doesn't cut off traffic until we know it's ok.
+			maxInflight = cfg.RaftMaxSizePerMsg * uint64(cfg.RaftMaxInflightMsgs) * 64
```

Comment:
Sadly, you need to worry about overflow here and stay at `MaxUint64` instead of hitting whatever random number you end up with wrap-around.

Reply:
Added some `big.Int` magic.